UCSLibraries/src/test/java/it/cnr/iit/AbstractPojoTest.java

```diff
@@ -0,0 +1,16 @@
+package it.cnr.iit;
+
+import static pl.pojo.tester.api.assertion.Assertions.assertPojoMethodsForAll;
+
+import org.junit.Test;
+
+import pl.pojo.tester.api.DefaultPackageFilter;
+import pl.pojo.tester.api.assertion.Method;
+
+public class AbstractPojoTest {
+
+    @Test
+    public void Should_Pass_All_Pojo_Tests() {

```

Comment:
first letter is capital? maybe sonar will complain about this => should_Pass_All_Pojo_Tests

Reply:
well spotted :-)
done